app/src/test/java/com/example/benne/daisyapp2/SmilParserUnitTests.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.benne.daisyapp2
+
+import com.example.benne.daisyapp2.data.daisy202.SmilParElement
+import com.example.benne.daisyapp2.data.daisy202.SmilParser.Companion.parseSmil
+import org.junit.BeforeClass
+import java.io.File
+import java.io.FileNotFoundException
+
+class SmilParserUnitTests {
+    companion object {
+        private val path = File("").absolutePath + "\\src\\test\\java\\com\\example\\benne\\daisyapp2"
```

Comment:
java seems to have a standard convention for resources
which is 
`src/pkg/resources`

- https://docs.gradle.org/current/userguide/building_java_projects.html#sec:java_resources
- https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

So we should try to follow that structure.